vaadin-login-flow-demo/src/main/java/com/vaadin/flow/component/login/vaadincom/LoginView.java

```diff
@@ -42,6 +42,8 @@ private void basicDemo() {
                 component.setError(true);
             }
         });
+
+        add(createLayout(component));
```

Comment:
I think you should add just `component` . The `createLayout` is not  exposed to users.